### PR TITLE
Sets up launch files for 3 cameras

### PR DESCRIPTION
**Pull request by ebretl**
- 3 cameras: camera_center, camera_left, camera_right
- names changed in urdf/tf/gazebo
- camera images are in /camera_{name}/image_color_rect

### CODE REVIEW

File: rr_platform/launch/camera_logitech_c615.launch [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    <!-- image size and type -->
+    <param name="width" value="640"/>
+    <param name="height" value="480"/>
+    <param name="video_mode" value="uncompressed"/> <!-- or yuyv/nv12/mjpeg -->
+    <param name="frame_rate" value="30"/>
+    <param name="timestamp_method" value="start"/>
+
+    <!-- image quality -->
+    <param name="auto_exposure" value="0"/>  <!-- Manual = 0, Auto = 3 -->
```
Comment (btdubs) [inline rr_platform/launch/camera_logitech_c615.launch]: I seem to remember some other parameter we changed that aperture priority vs. shutter priority vs something else. Feels like something we need to set here despite me not remembering what it was called right now.

**Comment by btdubs** [comment]
Can someone find time to refactor all of our launch files from using `/camera/image_color_rect` to `/camera_center/image_color_rect` so everything works still after merging this?

**Comment by btdubs** [comment]
Also, we probably need to suppress the number of topics published by this. Each camera now has like 20 topics because it has color, mono, and raw all with compressed, rectified, compressed and rectified, etc.

**Comment by ebretl** [comment]
> Can someone find time to refactor all of our launch files from using `/camera/image_color_rect` to `/camera_center/image_color_rect` so everything works still after merging this?

I think I did a large amount of this on the pose_tracking branch before school ended, but unfortunately those changes are entangled with other work on local mapping.

**Comment by btdubs** [comment]
Merged as part of larger PR for IARRC competition 2019 in #247

**Pull request closed without merging**